README.md

```diff
@@ -1,3 +1,7 @@
+## You probably shouldn't install Sydent. 
+
+Sydent isn't currently ready for widespread use. Feel free to use an alternative identity server implementation such as https://matrix.org/docs/projects/other/mxisd.html.
```

Comment:
Sydent isn't intended for DIY hosting yet because it lacks federated identity, mainly because we want to skip to better decentralised identity (although are running years late with this). Need to think about the wording of this some more.